### ISSUE TITLE
Add caching support

### ISSUE THREAD

**Issue by sebholstein** [sebholstein/astro-google-fonts-optimizer#1]
Prevent to download the same google fonts URL twice inside a single build run

**Comment by kerns** [comment]
Download @ build, ...I think this must be what next.js does?

**Comment by sebholstein** [comment]
@kerns yeah but I asked the astro team if there's any caching system built in, but there isn't atm. So ideally it wouldn't do the download for every build (when the assets are already in the cache).

**Comment by Herob527** [comment]
I published package that adds in-memory caching for downloaded cache and retrying fetch on failure

https://www.npmjs.com/package/@herob191/astro-google-fonts-optimizer

In case of problem with my  fork, post an issue there
https://github.com/Herob527/astro-google-fonts-optimizer
